massgov/mayflower
# DP-24261 [a11y] alert button label for screen reader users

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Clarify the accordion label and its state for the site wide alert for screen reader users by:
- removing **aria-label** 
- hiding the content of `span.ma__emergency-header__toggle__hide` and `span.ma__emergency-header__toggle__show`

The accordion state is announced to screen reader users with `aria-expanded`.  The button label shows the alert type and the title.  No more state indicating content is included to the title.

Tested in Drupal:  https://pr1411-pehlo5dntbrflq8mcx1of4rjsrlicfo3.tugboat.qa/

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-24261)
- [Github issue]()

## Steps to Test


1. Go to https://mayflower.digital.mass.gov/patternlab/?p=pages-location-park-content.
2. Check the markup for the alert button.
    - Find the button doesn't have `aria-label`.
    - Find the button content, `span.ma__emergency-header__toggle__hide` and `span.ma__emergency-header__toggle__show`, have `aria-hidden="true"`.

## Screenshots
<img width="895" alt="alert-button" src="https://user-images.githubusercontent.com/9633303/159297874-6424fb5a-9502-4ae7-be22-f730addaeb85.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
